extensions/amp-list/0.1/amp-list.js

```diff
@@ -642,30 +630,38 @@ export class AmpList extends AMP.BaseElement {
   }
 
   /**
-   * Converts the amp-list to de facto layout container. Must be called in
-   * mutation context.
+   * Converts the amp-list to de facto layout container. Called in mutate
+   * context.
+   * @return {!Promise}
    * @private
    */
   changeToLayoutContainer_() {
+    // TODO (#18875): cleanup resizable-children experiment
+    user().assert(isExperimentOn(this.win, 'amp-list-resizable-children'),
+        'Experiment amp-list-resizable-children is disabled');
+
     const previousLayout = this.element.getAttribute('layout');
     // If we have already changed to layout container, no need to run again.
     if (previousLayout == Layout.CONTAINER) {
-      return;
+      return Promise.resolve();
     }
-    this.undoPreviousLayout_(previousLayout);
-    this.container_.classList.remove(
-        'i-amphtml-fill-content',
-        'i-amphtml-replaced-content'
-    );
-    // The overflow element is generally hidden with visibility hidden,
-    // but after changing to layout container, this causes an undesirable
-    // empty white space so we hide it with display none instead.
-    const overflowElement = this.getOverflowElement();
-    if (overflowElement) {
-      toggle(overflowElement, false);
-    }
-    this.element.classList.add('i-amphtml-layout-container');
-    this.element.setAttribute('layout', 'container');
+
+    return this.mutateElement(() => {
+      this.undoPreviousLayout_(previousLayout);
```

Comment:
undoPreviousLayout_ switch statements needs a default case even if it's a fall through, based on js style guide.

default:
  // fall through.

https://google.github.io/styleguide/jsguide.html#features-switch-statements

Reply:
Good to know. Thank you!